additional debugs introduced due to VMXNET3 debugging (no functional changes)

--- a/grub-core/net/net.c	2017-04-04 12:26:09.136491213 +0200
+++ b/grub-core/net/net.c	2017-04-04 16:27:27.258430141 +0200
@@ -1450,13 +1450,21 @@
 receive_packets (struct grub_net_card *card, int *stop_condition)
 {
   int received = 0;
-  if (card->num_ifaces == 0)
+  if (card->num_ifaces == 0) {
+#ifdef M42_DEBUG
+    grub_printf("M42 DEBUG [recv_packets]: driver w/o ifaces\n");
+#endif
     return;
+  }
   if (!card->opened)
     {
       grub_err_t err = GRUB_ERR_NONE;
       if (card->driver->open)
 	err = card->driver->open (card);
+#ifdef M42_DEBUG
+      grub_printf("M42 DEBUG [recv_packets]: driver open card -> err=%d\n",
+		  err);
+#endif
       if (err)
 	{
 	  grub_errno = GRUB_ERR_NONE;
@@ -1496,12 +1504,29 @@
 {
   struct grub_net_card *card;
   grub_uint64_t start_time;
+
+#ifdef M42_DEBUG
+  grub_printf("M42 DEBUG [poll cards]: entry\n");
+#endif
+
   start_time = grub_get_time_ms ();
   while ((grub_get_time_ms () - start_time) < time
-	 && (!stop_condition || !*stop_condition))
+	 && (!stop_condition || !*stop_condition)) {
     FOR_NET_CARDS (card)
       receive_packets (card, stop_condition);
+  }
+
+#ifdef M42_DEBUG
+  grub_printf("M42 DEBUG [poll_cards]: exit from polling loop, %s\n",
+	      stop_condition==NULL ? "no stop condition backptr" :
+	      (*stop_condition ? "got stop condition" : "no stop condition"));
+#endif
+
   grub_net_tcp_retransmit ();
+
+#ifdef M42_DEBUG
+  grub_printf("M42 DEBUG [poll cards]: exit\n");
+#endif
 }
 
 static void
